src/dmd/attrib.h

```diff
@@ -248,7 +248,8 @@ class UserAttributeDeclaration : public AttribDeclaration
     Dsymbol *syntaxCopy(Dsymbol *s);
     Scope *newScope(Scope *sc);
     void setScope(Scope *sc);
-    Expressions *getAttributes();
     const char *kind() const;
     void accept(Visitor *v) { v->visit(this); }
 };
+
+Expressions* getAttributes(UserAttributeDeclaration uad);
```

Comment:
It seems that the stile is to have the * (star) right before the name of the function.

Reply:
Yes, and the param is a pointer too.

Reply:
Please add a test to cxx-unittest so that it is covered.

Reply:
@ibuclaw do you mean src/tests/cxxfrontend.c or should I add it to test/unit?

Reply:
> @ibuclaw do you mean src/tests/cxxfrontend.c or should I add it to test/unit?

cxxfrontend.c - just need to call the function from c++ to verify that mangling matches.